.circleci/config.yml

```diff
@@ -0,0 +1,117 @@
+version: 2.1
+
+references:
+  platform-docker: &docker
+    docker:
+    - image: objectrocket/platform-cicd:latest
```

Comment:
I would recommend changing this to `objectrocket/cicd:slim` as I plan on retiring platform-cicd docker image down the road

Reply:
but that platform-cicd image is perfectly valid until then, and well tested, it won't receive any new updates though. so up to you. You'd basically be the first one utilizing the cicd:slim image